### PR TITLE
feat: extract verification metadata

### PR DESCRIPTION
**Pull request by dzbo**
Move `verification` object to own type so it allow to import it by 3rd parties.

### CODE REVIEW

File: constants.ts [hunk 1]
```diff
@@ -120,14 +120,16 @@ export type LSP4DigitalAssetMetadata = {
   icon: ImageMetadata[];
 };
 
+export type MetadataVerification = {
```
Comment (richtera) [inline constants.ts:123]: But we have this type already and it's called just Verification

Comment (richtera) [inline constants.ts:123]: But we have this type already and it's called just Verification

Comment (dzbo) [reply constants.ts:123]: Where is that type?

Comment (richtera) [reply constants.ts:123]: https://github.com/lukso-network/tools-lsp-factory/blob/a9780fd2b80cf00f8ab4c2aa4910a890360cbabb/src/lib/interfaces/metadata.ts#L36
You're right it's in another repo.

Comment (dzbo) [reply constants.ts:123]: Yeah, it should be moved here. I will rename it tho

**Review by richtera — APPROVED** [review]

**Review by CJ42 — APPROVED** [review]

**Pull request merged**